### ISSUE TITLE
Map pageid to wikidata id

### ISSUE THREAD

**Issue by jcklie** [jcklie/wikimapper#4]
See https://stackoverflow.com/questions/38398893/is-there-any-dump-for-wikipedia-pageid-to-wikidata-id-mapping#

**Comment by CaydenPierce** [comment]
Agreed, this is my use case. Currently going from pageid -> title -> wikidataId. Would be worth having that built in.

Thanks for a great tool.
